components/table/demo/table-test.js

```diff
@@ -195,6 +195,16 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 		});
 	}
 
+	_loadItemsAsync() {
```

Comment:
This test element is already doing a lot, so I think it might be worth creating a new one -- even using `defineCE` in the test itself that just demonstrates the situation you want to test.